Shared/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -55,7 +56,8 @@ void CScriptDebugging::LogCustom(lua_State* luaVM, unsigned char ucRed, unsigned
     VSNPRINTF(szBuffer, MAX_STRING_LENGTH, szFormat, marker);
     va_end(marker);
 
-    LogString("", GetLuaDebugInfo(luaVM), szBuffer, 0, ucRed, ucGreen, ucBlue);
+    // Send empty SLuaDebugInfo if omitDebugInfo is true
+    LogString("", omitDebugInfo ? SLuaDebugInfo() : GetLuaDebugInfo(luaVM), szBuffer, 0, ucRed, ucGreen, ucBlue);
```

Comment:
Instantiating an empty `SLuaDebugInfo` to act as if we were not sending debug info looks very hacky-ish to me.

Reply:
Well, the function takes a reference, so I can't use a `nullptr` and I'd rather avoid redefining it to use a pointer. `SLuaDebugInfo` constructor actually produces a pretty well defined no-debug-info object. https://github.com/multitheftauto/mtasa-blue/blob/0d0d60c7df86413d03066903549c565083a716c4/Client/mods/deathmatch/logic/lua/LuaCommon.h#L93 https://github.com/multitheftauto/mtasa-blue/blob/0d0d60c7df86413d03066903549c565083a716c4/Server/mods/deathmatch/logic/lua/LuaCommon.h#L87

I could change it to something like 
```cpp
SLuaDebugInfo debugInfo;
if (!omitDebugInfo)
    debugInfo = GetLuaDebugInfo(luaVM);
    
LogString("", debugInfo, szBuffer, 0, ucRed, ucGreen, ucBlue);
```
if that looks less hackish. Thoughts?